### DISCUSSION TITLE
How do i remove physics from a mesh

### DISCUSSION THREAD

**Discussion by webciter** [enable3d/enable3d#189] [Help / Questions]
Whats the correct way of removing physics from a THREE.Mesh, i'v tried physics.destroy(mesh.body) but the debug draw still shows the physics outline.

**Comment by yandeu** [comment]
This is the correct way. See https://enable3d.io/examples/create-and-destroy-objects-and-physics.html,

Maybe some outlines get stuck in the debug drawer? I don't know :/

**Reply by webciter** [reply]
i'v found that because i was using mesh.clone(), this function does not work. Then i later set the Mesh.name to a unique value, which now makes it work.

**Reply by yandeu** [reply]
`Mesh.clone()` does not work?

Can you try to use the `ExtendedMesh` instead?
